frontend/awx/administration/instances/hooks/useSelectPeers.tsx

```diff
@@ -0,0 +1,56 @@
+import { useCallback, useMemo } from 'react';
+import { usePageDialog } from '../../../../../framework';
+import { awxAPI } from '../../../common/api/awx-utils';
+import { useAwxView } from '../../../common/useAwxView';
+import { Instance } from '../../../interfaces/Instance';
+import { usePeersFilters } from '../Instances';
+import { usePeersColumns } from './usePeersColumns';
+import { MultiSelectDialog } from '../../../../../framework';
+import { useTranslation } from 'react-i18next';
+
+function SelectPeers(props: {
+  isLookup: boolean;
+  title: string;
+  onSelect: (instances: Instance[]) => void;
+}) {
+  const toolbarFilters = usePeersFilters();
+  const tableColumns = usePeersColumns();
```

Comment:
```suggestion
  const tableColumns = useInstancesColumns({ disableLinks: true });
```

We can reuse the instance columns hook here and just omit a few things instead of creating another hook to handle this table view.